README.md

```diff
@@ -43,7 +43,7 @@ In order to run a StackExchange account as a bot, you need to follow the followi
 3. Create a StackExchange account on http://stackexchange.com.
 4. Make sure you can log in to it.
 5. Create an account on a specific site, for example http://stackoverflow.com.
-6. Earn 20 reputation, following the rules of the particular site.
+6. Earn 20 reputation, following the rules of the particular site. (or you can make your own chatroom and give the bot exclusive write access)
```

Comment:
that's incorrect. The user-interface there is a little confusing, but explicit write access for users with less than 20 rep can only be granted by diamond moderators.  RoomOwners do not have that privilege